### ISSUE TITLE
Iconfont new version is it compatible with FontExplorer?

### ISSUE THREAD

**Issue by misterfulanito** [keremciu/sketch-iconfont#81]
I've been trying to reinstall your amazing plugin with no luck, any icon font is not detected by the plugin if I using FontExplorer, do you have any ideas?

Thank you guys.

**Comment by keremciu** [comment]
Hello @misterfulanito,

What is the problem with this plugin? Can you explain more?

**Comment by misterfulanito** [comment]
Sure, I usually use FontExplorer for activate/deactivate my fonts and I'd been using it until the last Iconfont plugin version but something happened and Iconfont can't recognize my icon fonts through FontExplorer.

I'm able to work with the icon fonts with Sketch but the plugin is not showing up their magic.

FontExplorer: Font(s) are activated, I activated only one for this demo
![captura de pantalla 2016-11-23 a la s 09 12 03](https://cloud.githubusercontent.com/assets/813025/20567157/59b23ee6-b15d-11e6-8085-0be5b3f87be3.png)

Iconfont can't detect it on Sketch
![captura de pantalla 2016-11-23 a la s 09 11 51](https://cloud.githubusercontent.com/assets/813025/20567156/59adce1a-b15d-11e6-889d-0b3be5aac54d.png)

I already tried to reinstall it some times with no luck.

Thank you @keremciu for your help.
